clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClientSpec.groovy

```diff
@@ -52,6 +52,58 @@ class DockerRegistryClientSpec extends Specification {
     TypedInput catalogTypedInput = new TypedByteArray("application/json", json.getBytes())
     Response catalogResponse = new Response("/v2/_catalog/",200, "nothing", Collections.EMPTY_LIST, catalogTypedInput)
     getCatalog(_,_,_) >> catalogResponse
+
+    String schemaJson = "{\n" +
```

Comment:
Not a big deal, but I think you can write this sample JSON more easily using triple-quote strings: https://groovy-lang.org/syntax.html#_triple_single_quoted_string